src/core/components/auth/authorization-popup.jsx

```diff
@@ -8,6 +8,13 @@ export default class AuthorizationPopup extends React.Component {
     authActions.showDefinitions(false)
   }
 
+  onKeyDown = (e) => {
```

Comment:
**Change request**: I'd like to have a test that ensures `this.close` is called when Esc is pressed, and another that ensures other keydowns don't trigger `this.close`.

@smhigley, in our discussion I believe we landed on creating a synthetic keydown event in a component unit test being best. That'd be great, but feel free to take a different approach (e.g., add an end-to-end test case to our Cypress test suite) if you have trouble setting such a test up with Enzyme. Happy to provide pointers, just let me know!